docs/README.md

```diff
@@ -1,29 +1,232 @@
-# User Guide
+# Duke Chatbot
+Duke chatbot is a java program that helps you to manage your tasks
```

Comment:
I really like how you've added to the documentation. It sort of acts as a user guide, which is so useful. It's definitely easy to understand the product with this!